docs/cli-client/service/bind.md

```diff
@@ -7,29 +7,28 @@ Create a new service binding
-| --def-chain-id        |                         | [string] the ID of the blockchain defined of the service                                                                                              |  Yes     |
-| --deposit             |                         | [string] deposit of binding                                                                                                                           |  Yes     |
-| --prices              |                         | [strings] prices of binding, will contains all method                                                                                                 |          |
-| --service-name        |                         | [string] service name                                                                                                                                 |  Yes     |
-| --usable-time         |                         | [int] an integer represents the number of usable service invocations per 10,000                                                                       |  Yes     |
+| --avg-rsp-time        |                         | the average service response time in milliseconds                                                                                               |  Yes     |
+| --bind-type           |                         | type of binding, valid values can be Local and Global                                                                                        |  Yes     |
+| --def-chain-id        |                         | the ID of the blockchain defined of the service                                                                                              |  Yes     |
+| --deposit             |                         | deposit of binding                                                                                                                           |  Yes     |
+| --prices              |                         | prices of binding, will contains all method                                                                                                 |          |
+| --service-name        |                         | service name                                                                                                                                 |  Yes     |
+| --usable-time         |                         | an integer represents the number of usable service invocations per 10,000                                                                       |  Yes     |
 
 ## Examples
 
 ### Add a binding to an existing service definition
-In service binding, you need to define `deposit`, minimum deposit = `price` of this service * `MinDepositMultiple` in genesis file
-
+In service binding, you need to define `deposit`,  the minimum mortgage amount of this `deposit` is `price` * `MinDepositMultiple` (defined in genesis file)
```

Comment:
`,  ` -> `, `

Reply:
defined by system parameters, can be modified through governance

Reply:
fixed